### PR TITLE
Add landing pages

### PR DESCRIPTION
**Pull request by VadymSachenko**
## PR Description
This PR adds missing landing pages to the folders in `/docs/dev/`
Jira task: https://spryker.atlassian.net/browse/DOC-65
TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/scos/dev/back-end-development/data-manipulation/data-publishing/data-publishing.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Data publishing
+description: This page lists documents inside the Data publishing section
+last_updated: Apr 25, 2023
+template: howto-guide-template
+---
+
+The *Data publishing* section includes the following documents:
```
Comment (lenadoc) [inline docs/scos/dev/back-end-development/data-manipulation/data-publishing/data-publishing.md:8]: Can you add a few words on what this section is about?

File: docs/scos/dev/back-end-development/extend-spryker/extend-spryker.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: Extend Spryker
+description: This section lists documents inside the Extend Spryker section.
+last_updated: Apr 26, 2023
+template: concept-topic-template
+---
+
+The *Extend Spryker* section includes the following documents:
```
Comment (lenadoc) [inline docs/scos/dev/back-end-development/extend-spryker/extend-spryker.md:8]: Can you add a few words on what this section is about?

File: docs/scos/dev/back-end-development/extend-spryker/spryker-os-module-customisation/spryker-os-module-customisation.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+title: Spryker OS module customisation
+description: This section lists documents inside the Spryker OS module customisation section.
+last_updated: Apr 26, 2023
+template: concept-topic-template
+---
+
+The *Spryker OS module customisation* section includes the following documents:
```
Comment (lenadoc) [inline docs/scos/dev/back-end-development/extend-spryker/spryker-os-module-customisation/spryker-os-module-customisation.md:8]: Can you add a few words on what this section is about?

File: docs/scos/dev/guidelines/testing-guidelines/testing-guidelines.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+title: Testing guidelines
+description: This section lists documents inside the Testing guidelines section.
+last_updated: Apr 26, 2023
+template: concept-topic-template
+---
+
+The *Testing guidelines* section includes the following documents:
```
Comment (lenadoc) [inline docs/scos/dev/guidelines/testing-guidelines/testing-guidelines.md:8]: Can you add a few words on what this section is about?

File: docs/scos/dev/migration-concepts/silex-replacement/router/router.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+---
+title: Router
+description: This section lists documents inside the Router section.
+last_updated: Apr 26, 2023
+template: concept-topic-template
+---
+
+The *Router* section includes the following documents:
```
Comment (lenadoc) [inline docs/scos/dev/migration-concepts/silex-replacement/router/router.md:8]: Can you add a few words on what this section is about?

File: docs/scos/dev/sdk/zed-api/zed-api.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
+---
+title: Zed API
+description: This section lists documents inside the Zed API section.
+last_updated: Apr 26, 2023
+template: concept-topic-template
+---
+
+The *Zed API* section includes the following documents:
```
Comment (lenadoc) [inline docs/scos/dev/sdk/zed-api/zed-api.md:8]: Can you add a few words on what this section is about?

File: docs/scos/dev/technology-partner-guides/202212.0/marketing-and-conversion/analytics/fact-finder/fact-finder.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
+---
+title: FACT-Finder
+description: This section lists documents inside the FACT-Finder section.
+last_updated: Apr 26, 2023
+template: concept-topic-template
+---
+
+The *FACT-Finder* section includes the following documents:
```
Comment (lenadoc) [inline docs/scos/dev/technology-partner-guides/202212.0/marketing-and-conversion/analytics/fact-finder/fact-finder.md:8]: Can you add a few words on what this section is about?

File: docs/scos/dev/technology-partner-guides/202212.0/marketing-and-conversion/customer-communication/episerver/episerver.md [hunk 8]
```diff
@@ -0,0 +1,12 @@
+---
+title: Episerver
+description: This section lists documents inside the Episerver section.
+last_updated: Apr 26, 2023
+template: concept-topic-template
+---
+
+The *Episerver* section includes the following documents:
```
Comment (lenadoc) [inline docs/scos/dev/technology-partner-guides/202212.0/marketing-and-conversion/customer-communication/episerver/episerver.md:8]: Can you add a few words on what this section is about?

File: docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/feature-howtos.md [hunk 9]
```diff
@@ -0,0 +1,10 @@
+---
+title: Feature HowTos
+description: This section lists documents inside the Feature HowTos section.
+last_updated: Apr 26, 2023
+template: concept-topic-template
+---
+
+The *Feature HowTos* section includes the following documents:
```
Comment (lenadoc) [inline docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/feature-howtos.md:8]: Can you add a few words on what this section is about?

File: docs/scos/dev/tutorials-and-howtos/introduction-tutorials/introduction-tutorials.md [hunk 10]
```diff
@@ -0,0 +1,13 @@
+---
+title: Introduction Tutorials
+description: This section lists documents inside the Introduction Tutorials section.
+last_updated: Apr 26, 2023
+template: concept-topic-template
+---
+
+The *Introduction Tutorials* section includes the following documents:
```
Comment (lenadoc) [inline docs/scos/dev/tutorials-and-howtos/introduction-tutorials/introduction-tutorials.md:8]: Can you add a few words on what this section is about?

**Review by lenadoc — CHANGES_REQUESTED** [review]

File: docs/scos/dev/back-end-development/data-manipulation/data-publishing/data-publishing.md (hunk 1 above)
Comment (VadymSachenko) [inline docs/scos/dev/back-end-development/data-manipulation/data-publishing/data-publishing.md:8]:
```suggestion
The *Data publishing* section describes the functionalities and processes related to data publishing. Learn how to add publish events for Publish and Synchronization, understand the debugging listeners using the event:trigger:listener command, and implement event trigger publisher plugins. Additionally, explore the implementation of Publish and Synchronization in your code, synchronization plugins, and the synchronization process for multi-store shop systems. Discover how to perform repeated exports, regenerate published data, and enable multiple mappings to retrieve data without specifying the resource's ID. 

In this section, you can find the following documents:
```

File: docs/scos/dev/back-end-development/extend-spryker/extend-spryker.md (hunk 2 above)
Comment (VadymSachenko) [reply docs/scos/dev/back-end-development/extend-spryker/extend-spryker.md:8]:
```suggestion
The *Extend Spryker* section section provides guidance on extending Spryker to enhance its functionality. In this section, you can find documents about creating modules for encapsulating new concepts and learn about extending a core module used by another core module. You explore different approaches such as replacement class, inheritance object, and composition to extend the core. Also, this section shows how to extend the Spryker Core functionality, use the Spryker Engine and learn how to install the module structure and configuration.

In this section, you can find the following documents and subsections:
```
```

File: docs/scos/dev/back-end-development/extend-spryker/spryker-os-module-customisation/spryker-os-module-customisation.md (hunk 3 above)
Comment (VadymSachenko) [reply docs/scos/dev/back-end-development/extend-spryker/spryker-os-module-customisation/spryker-os-module-customisation.md:8]:
```suggestion
Learn to customize modules in Spryker OS to meet your specific needs by extending core modules and exploring different approaches. Understand the folder structure used in Spryker Commerce OS and install module structure for project-level customization.

This section includes the following documents:
```

File: docs/scos/dev/guidelines/testing-guidelines/testing-guidelines.md (hunk 4 above)
Comment (VadymSachenko) [reply docs/scos/dev/guidelines/testing-guidelines/testing-guidelines.md:8]:
```suggestion
Get comprehensive guidance for effective testing in Spryker projects. Discover a range of topics, including available test helpers, code coverage generation, data builders for creating transfer objects, executing tests, setting up efficient test organization, configuring and using the Codeception test framework, testing best practices and console commands, and more.

The *Testing guidelines* section includes the following documents:
```

File: docs/scos/dev/migration-concepts/silex-replacement/router/router.md (hunk 5 above)
Comment (VadymSachenko) [reply docs/scos/dev/migration-concepts/silex-replacement/router/router.md:8]: In this section, you can find guidance on setting up and configuring the router in both Yves and Zed. You can learn how to install and integrate the router extension module in Yves, as well as install and configure the router in Zed.

For more information, see the following document:

Comment (VadymSachenko) [reply docs/scos/dev/migration-concepts/silex-replacement/router/router.md:8]:
```suggestion
In this section, you can find guidance on setting up and configuring the router in both Yves and Zed. You can learn how to install and integrate the router extension module in Yves, as well as install and configure the router in Zed.

For more information, see the following document:
```

File: docs/scos/dev/sdk/zed-api/zed-api.md (hunk 6 above)
Comment (VadymSachenko) [reply docs/scos/dev/sdk/zed-api/zed-api.md:8]:
```suggestion
Explore the flexible REST API offered by Spryker OS for Zed and the basic functionality of mapping REST actions to the backend persistence layer. Learn how to configure the API and explore the Zed API processor stack and the project implementation process for activating API and API bundles. Also, expand your knowledge about Zed API resources to leverage their capabilities effectively.

For more detailed information, see the following documents:
```

File: docs/scos/dev/setup/installing-spryker-with-docker/configuration/configuration.md [hunk 11]
```diff
@@ -0,0 +1,11 @@
+---
+title: Configuration
+description: This section lists documents inside the Configuration section.
+last_updated: Apr 26, 2023
+template: concept-topic-template
+---
+
+The *Configuration* section includes the following documents:
```
[the remark this answers is not in the document]
Comment (VadymSachenko) [reply docs/scos/dev/setup/installing-spryker-with-docker/configuration/configuration.md:8]:
```suggestion
The "Configuration" section provides valuable guidance on configuring your Spryker setup. Learn how to adjust Jenkins for a Docker environment and how to configure Spryker after installation with Docker. Additionally, explore the process of setting up a self-signed SSL certificate to enable secure access to your instance.

In this section you can find the following documents:
```

File: docs/scos/dev/technology-partner-guides/202212.0/marketing-and-conversion/analytics/fact-finder/fact-finder.md (hunk 7 above)
Comment (VadymSachenko) [reply docs/scos/dev/technology-partner-guides/202212.0/marketing-and-conversion/analytics/fact-finder/fact-finder.md:8]:
```suggestion
The "Fact-Finder" section provides guidance on integrating and utilizing the FACT-Finder module. Learn how to export product data, install and configure web components and the NG API, integrate Fact Finder modules, and leverage campaigns, recommendations, search suggestions, and tracking for improved search functionality and personalized user experiences.

For detailed instructions, see the respective documents:
```

File: docs/scos/dev/technology-partner-guides/202212.0/marketing-and-conversion/customer-communication/episerver/episerver.md (hunk 8 above)
Comment (VadymSachenko) [reply docs/scos/dev/technology-partner-guides/202212.0/marketing-and-conversion/customer-communication/episerver/episerver.md:8]:
```suggestion
The "Episerver" section provides guidance on integrating and utilizing Episerver. Learn about the Episerver API requests and order reference commands. Follow the instructions to integrate and configure Episerver in your project.

For more information the following documents in this section:
```

File: docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/feature-howtos.md (hunk 9 above)
Comment (VadymSachenko) [reply docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/feature-howtos.md:8]:
```suggestion
The "Feature HowTos" section provides step-by-step instructions about adding support for number formatting in both the Storefront and Back Office user interfaces. To integrate and implement number formatting, follow the detailed instructions in this section:
```

File: docs/scos/dev/tutorials-and-howtos/introduction-tutorials/introduction-tutorials.md [hunk 12]
```diff
@@ -0,0 +1,13 @@
+---
+title: Introduction Tutorials
```
Comment (VadymSachenko) [inline docs/scos/dev/tutorials-and-howtos/introduction-tutorials/introduction-tutorials.md:2]:
```suggestion
title: Introduction tutorials
```

File: docs/scos/dev/setup/installing-spryker-with-docker/configuration/configuration.md [hunk 13]
```diff
@@ -0,0 +1,13 @@
+---
+title: Configuration
+description: This section lists documents inside the Configuration section.
+last_updated: Apr 26, 2023
+template: concept-topic-template
+---
+
+The "Configuration" section provides guidance about configuring your Spryker setup. Learn how to adjust Jenkins for a Docker environment and how to configure Spryker after installation with Docker. Additionally, explore the process of setting up a self-signed SSL certificate to enable secure access to your instance.
```
Comment (VadymSachenko) [inline docs/scos/dev/setup/installing-spryker-with-docker/configuration/configuration.md:8]:
```suggestion
The *Configuration* section provides guidance about configuring your Spryker setup. Learn how to adjust Jenkins for a Docker environment and how to configure Spryker after installation with Docker. Additionally, explore the process of setting up a self-signed SSL certificate to enable secure access to your instance.
```

File: docs/scos/dev/migration-concepts/silex-replacement/router/router.md [hunk 14]
```diff
@@ -0,0 +1,12 @@
+---
+title: Router
+description: This section lists documents inside the Router section.
+last_updated: Apr 26, 2023
+template: concept-topic-template
+---
+
+In this section, you can find guidance on setting up and configuring the router in both Yves and Zed. You can learn how to install and integrate the router extension module in Yves, as well as install and configure the router in Zed.
```
Comment (VadymSachenko) [inline docs/scos/dev/migration-concepts/silex-replacement/router/router.md:8]:
```suggestion
In this section, you can find guidance abouot setting up and configuring the router in both Yves and Zed. You can learn how to install and integrate the router extension module in Yves, as well as install and configure the router in Zed.
```

File: docs/scos/dev/technology-partner-guides/202212.0/marketing-and-conversion/customer-communication/episerver/episerver.md [hunk 15]
```diff
@@ -0,0 +1,14 @@
+---
+title: Episerver
+description: This section lists documents inside the Episerver section.
+last_updated: Apr 26, 2023
+template: concept-topic-template
+---
+
+The "Episerver" section provides guidance about integrating and utilizing Episerver. Learn about the Episerver API requests and order reference commands. Follow the instructions to integrate and configure Episerver in your project.
```
Comment (VadymSachenko) [inline docs/scos/dev/technology-partner-guides/202212.0/marketing-and-conversion/customer-communication/episerver/episerver.md:8]:
```suggestion
The *Episerver* section provides guidance about integrating and utilizing Episerver. Learn about the Episerver API requests and order reference commands. Follow the instructions to integrate and configure Episerver in your project.
```

File: docs/scos/dev/architecture/module-api/module-api.md [hunk 16]
```diff
@@ -0,0 +1,13 @@
+---
+title: Module API
+description: Learn about Module APIs, performance and scalability, semantic versioning, and Composer constrains for customer modules.
+last_updated: Apr 25, 2023
+template: concept-topic-template
+---
+
+In this section, you can find the following documents:
```
Comment (VadymSachenko) [inline docs/scos/dev/architecture/module-api/module-api.md:8]:
```suggestion
In this section, you can learn about module APIs, performance and scalability, semantic versioning, and Composer constraints for customer modules. For specific information, see these documents:
```

File: docs/scos/dev/back-end-development/data-manipulation/data-ingestion/structural-preparations/structural-preparations.md [hunk 17]
```diff
@@ -0,0 +1,12 @@
+---
+title: Data ingestion
```
Comment (VadymSachenko) [inline docs/scos/dev/back-end-development/data-manipulation/data-ingestion/structural-preparations/structural-preparations.md:2]:
```suggestion
title: Structural preparation
```

File: docs/scos/dev/back-end-development/data-manipulation/data-ingestion/structural-preparations/structural-preparations.md [hunk 18]
```diff
@@ -0,0 +1,12 @@
...
+description: This page lists documents inside the Structural preparations section
+last_updated: Apr 25, 2023
+template: howto-guide-template
+---
+
+The *Structural preparations* section includes the following documents:
```
Comment (VadymSachenko) [inline docs/scos/dev/back-end-development/data-manipulation/data-ingestion/structural-preparations/structural-preparations.md:8]:
```suggestion
In the *Structural preparations" section, you can find information about various aspects of setting up and organizing your project's structure. Here, you can learn how to create, use, and expand transfer objects effectively for data manipulation as well as how to extend the existing database schema. Also, you can explore the integration with the `thephpleague` flysystem vendor package.

The *Structural preparations* section includes the following documents:
```

File: docs/scos/dev/architecture/module-api/module-api.md [hunk 19]
```diff
@@ -0,0 +1,13 @@
+---
+title: Module API
+description: Learn about Module APIs, performance and scalability, semantic versioning, and Composer constrains for customer modules.
+last_updated: Apr 25, 2023
+template: concept-topic-template
+---
+
+In this section, you can learn about module APIs, performance and scalability, semantic versioning, and Composer constraints for customer modules. For specific information, see these documents:
```
Comment (lenadoc) [inline docs/scos/dev/architecture/module-api/module-api.md:8]:
```suggestion
This section is about module APIs, performance and scalability, semantic versioning, and Composer constraints for customer modules. For specific information, see these documents:
```

File: docs/scos/dev/back-end-development/data-manipulation/data-ingestion/structural-preparations/structural-preparations.md [hunk 20]
```diff
@@ -0,0 +1,14 @@
+---
+title: Structural preparation
+description: This page lists documents inside the Structural preparations section
+last_updated: Apr 25, 2023
+template: howto-guide-template
+---
+
+In the *Structural preparations" section, you can find information about various aspects of setting up and organizing your project's structure. Here, you can learn how to create, use, and expand transfer objects effectively for data manipulation as well as how to extend the existing database schema. Also, you can explore the integration with the `thephpleague` flysystem vendor package.
```
Comment (lenadoc) [inline docs/scos/dev/back-end-development/data-manipulation/data-ingestion/structural-preparations/structural-preparations.md:8]:
```suggestion
The *Structural preparation* section covers information about various aspects of setting up and organizing your project's structure. Here, you will learn how to create, use, and expand transfer objects effectively for data manipulation as well as how to extend the existing database schema. Also, you can explore the integration with the `thephpleague` flysystem vendor package.
```

File: docs/scos/dev/sdk/zed-api/zed-api.md [hunk 21]
```diff
@@ -0,0 +1,16 @@
+---
+title: Zed API
+description: This section lists documents inside the Zed API section.
+last_updated: Apr 26, 2023
+template: concept-topic-template
+---
+
+Explore the flexible REST API offered by Spryker OS for Zed and the basic functionality of mapping REST actions to the backend persistence layer. Learn how to configure the API and explore the Zed API processor stack and the project implementation process for activating API and API bundles. Also, expand your knowledge about Zed API resources to leverage their capabilities effectively.
```
Comment (lenadoc) [inline docs/scos/dev/sdk/zed-api/zed-api.md:8]:
```suggestion
Explore the flexible REST API offered by the Spryker OS for Zed and the basic functionality of mapping REST actions to the backend persistence layer. Learn how to configure the API and explore the Zed API processor stack and the project implementation process for activating API and API bundles. Also, expand your knowledge about Zed API resources to leverage their capabilities effectively.
```

File: docs/scos/dev/setup/installing-spryker-with-docker/docker-installation-prerequisites/installing-docker-prerequisites.md [hunk 22]
```diff
@@ -0,0 +1,16 @@
+---
+title: Installing Docker prerequisites
+description: This section
+last_updated: Mar 30, 2023
+template: howto-guide-template
+---
+
+This section of the Spryker documentation describes the prerequisites for installing Spryker in Docker on Linux, MacOS, and Windows.
```
Comment (lenadoc) [inline docs/scos/dev/setup/installing-spryker-with-docker/docker-installation-prerequisites/installing-docker-prerequisites.md:8]:
```suggestion
This section describes the prerequisites for installing Spryker in Docker on Linux, MacOS, and Windows.
```

File: docs/scos/dev/technology-partner-guides/202212.0/marketing-and-conversion/analytics/fact-finder/fact-finder.md [hunk 23]
```diff
@@ -0,0 +1,20 @@
+---
+title: FACT-Finder
+description: This section lists documents inside the FACT-Finder section.
+last_updated: Apr 26, 2023
+template: concept-topic-template
+---
+
+The *Fact-Finder* section provides guidance about integrating and utilizing the FACT-Finder module. Learn how to export product data, install and configure web components and the NG API, integrate Fact Finder modules, and leverage campaigns, recommendations, search suggestions, and tracking for improved search functionality and personalized user experiences.
```
Comment (lenadoc) [inline docs/scos/dev/technology-partner-guides/202212.0/marketing-and-conversion/analytics/fact-finder/fact-finder.md:8]:
```suggestion
The *Fact-Finder* section provides guidance about integrating and utilizing the FACT-Finder module.
```

File: docs/scos/dev/technology-partner-guides/202212.0/marketing-and-conversion/customer-communication/episerver/episerver.md [hunk 24]
```diff
@@ -0,0 +1,14 @@
+---
+title: Episerver
+description: This section lists documents inside the Episerver section.
+last_updated: Apr 26, 2023
+template: concept-topic-template
+---
+
+The *Episerver* section provides guidance about integrating and utilizing Episerver. Learn about the Episerver API requests and order reference commands. Follow the instructions to integrate and configure Episerver in your project.
```
Comment (lenadoc) [inline docs/scos/dev/technology-partner-guides/202212.0/marketing-and-conversion/customer-communication/episerver/episerver.md:8]:
```suggestion
The *Episerver* section provides guidance about integrating, configuring, and utilizing Episerver.
```

File: docs/scos/dev/technology-partner-guides/202212.0/marketing-and-conversion/customer-communication/inxmail/inxmail.md [hunk 25]
```diff
@@ -0,0 +1,12 @@
+---
+title: Inxmail
+description: This section lists documents inside the Inxmail section.
+last_updated: Apr 26, 2023
+template: concept-topic-template
+---
+
```
Comment (lenadoc) [inline docs/scos/dev/technology-partner-guides/202212.0/marketing-and-conversion/customer-communication/inxmail/inxmail.md:7]: short description? :)

File: docs/scos/dev/technology-partner-guides/202212.0/marketing-and-conversion/personalization-and-cross-selling/econda/econda.md [hunk 26]
```diff
@@ -0,0 +1,13 @@
+---
+title: Econda
+description: This section lists documents inside the Econda section.
+last_updated: Apr 26, 2023
+template: concept-topic-template
+---
+
```
Comment (lenadoc) [inline docs/scos/dev/technology-partner-guides/202212.0/marketing-and-conversion/personalization-and-cross-selling/econda/econda.md:7]: no short description here as well

File: docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/feature-howtos.md [hunk 27]
```diff
@@ -0,0 +1,10 @@
+---
+title: Feature HowTos
+description: This section lists documents inside the Feature HowTos section.
+last_updated: Apr 26, 2023
+template: concept-topic-template
+---
+
```
Comment (lenadoc) [inline docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/feature-howtos.md:7]: Consider moving these two HowTos to respective PBCs

**Review by lenadoc — CHANGES_REQUESTED** [review]

File: docs/scos/dev/migration-concepts/silex-replacement/router/router.md [hunk 28]
```diff
@@ -0,0 +1,12 @@
+---
+title: Router
+description: This section lists documents inside the Router section.
+last_updated: Apr 26, 2023
+template: concept-topic-template
+---
+
+In this section, you can find guidance abouot setting up and configuring the router in both Yves and Zed. You can learn how to install and integrate the router extension module in Yves, as well as install and configure the router in Zed.
```
Comment (VadymSachenko) [inline docs/scos/dev/migration-concepts/silex-replacement/router/router.md:8]:
```suggestion
In this section, you can find guidance about setting up and configuring the router in both Yves and Zed. You can learn how to install and integrate the router extension module in Yves, as well as install and configure the router in Zed.
```

File: docs/scos/dev/technology-partner-guides/202212.0/marketing-and-conversion/personalization-and-cross-selling/econda/econda.md [hunk 29]
```diff
@@ -0,0 +1,13 @@
...
+The *Econda* section includes the following documents:
```
Comment (VadymSachenko) [inline docs/scos/dev/technology-partner-guides/202212.0/marketing-and-conversion/personalization-and-cross-selling/econda/econda.md:8]:
```suggestion
In this section, you can learn how to install and integrate Econda, export its data, integrate Econda cross-selling, and add the Econda tracking code into Spryker Commerce OS. For detailed guidance, see the following documents:


Integrating
```

File: docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/feature-howtos.md (hunk 27 above)
Comment (VadymSachenko) [reply docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/feature-howtos.md:7]: Out of scope of this task

File: docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/feature-howtos.md [hunk 30]
```diff
@@ -0,0 +1,10 @@
...
+The *Feature HowTos* section provides step-by-step instructions about adding support for number formatting in both the Storefront and Back Office user interfaces. To integrate and implement number formatting, follow the detailed instructions in this section:
+* ["HowTo: Add support of number formatting in the Back Office"](/docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/howto-add-support-of-number-formatting-in-the-back-office.html)
+* ["HowTo: Display custom names for order item states in the Storefront"](/docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/howto-add-support-for-number-formatting-in-the-storefront.html)
```
Comment (VadymSachenko) [inline docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/feature-howtos.md:10]:
```suggestion
* ["HowTo: Display custom names for order item states in the Storefront"](/docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/howto-add-support-for-number-formatting-in-the-storefront.html)
```

**Review by lenadoc — DISMISSED** [review]

**Pull request merged**